src/wasm-binary.h

```diff
@@ -375,8 +375,12 @@ enum EncodedType {
   // run-time type info type, with depth index n
   rtt_n = -0x17, // 0x69
   // run-time type info type, without depth index n
-  rtt = -0x18,     // 0x68
-  dataref = -0x19, // 0x67
+  rtt = -0x18,              // 0x68
+  dataref = -0x19,          // 0x67
```

Comment:
Add some comments between `rtt` and `dataref`? This looks all these ref types belong to "run-time type info type, without depth index n" category.